crates/risc0-bonsai/src/host.rs

```diff
@@ -331,22 +331,15 @@ impl<'a> Risc0BonsaiHost<'a> {
         // Prepare input data and upload it.
         let client = self.client.as_ref().unwrap();
 
-        let mut input_data = vec![];
-        let mut buf = borsh::to_vec(&item).unwrap();
-        // append [0..] alignment
+        let mut buf = borsh::to_vec(&item).expect("Risc0 hint serialization is infallible");
+        // append [0..] alignment to cast &[u8] to &[u32]
         let rem = buf.len() % 4;
         if rem > 0 {
```

Comment:
I am not sure if we need this anymore

Reply:
IMO it should be kept like this. Even if it doesn't benefit us anymore, if we were to ever send more data for any reason, it could be good to have the pre-knowledge that every message is aligned properly.

Reply:
But... next reads will always fail as there will be dummy 0s. Hence with the reader we either find and read those alignment zeroes after every read, which doesn't seem good to me tbh, or we actually remove alignment. I will still test, but probably will go for 2nd.

Reply:
went with the 2nd option, no more alignments on host.